managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -326,6 +327,11 @@ public enum PositionBound {
      */
     final ConcurrentLinkedQueue<OpAddEntry> pendingAddEntries = new ConcurrentLinkedQueue<>();
 
+    private volatile ManagedLedgerAttributes managedLedgerAttributes;
```

Comment:
I would just set this field as final and initialize it in constructor (even if otel is not used yet)